### DISCUSSION TITLE
Allow sorting scores by the date they were set

### DISCUSSION THREAD

**Discussion by memesnightmares** [ppy/osu#14468] [Ideas] [answered]
This isn't super useful for global/country/friend leaderboards, but for looking at all of the scores that I've set on a map (local scores tab) I frequently want to see my past scores in chronological order i.e. to see my first play on a map, or maybe to see how quickly I've improved over time. It would be very nice to have a toggle setting to change the sort from "best" to "chronological". I don't feel strongly about any particular design for this feature, it could be a global setting in the settings menu or it could be a toggle similar to the "Selected Mods" toggle.

**Comment by peppy** [comment]
Sounds like a good idea, for sure.

**Reply by memesnightmares** [reply]
Could you create a tracking issue like you did for #14469? Or would this sort of feature request be tracked differently then that more minor request?

**Reply by peppy** [reply]
this discussion is already a request.

**Reply by ArchUsr64** [reply]
Similar sort functionality for scores based on accuracy, max combo and misses could also be a useful feature.

**Comment by memesnightmares** [comment]
Was just thinking about this again, I'd like to add the concept of having some kind of "recently played" view that shows all your plays in chronological order (even across maps). Global score sorts like that could also be cool for any stats that are relevant across maps i.e. pp, maybe acc. But I don't want to expand the scope of this feature request too far :P

**Comment by Joehuu** [comment, marked as answer]
Will be added in https://github.com/ppy/osu/pull/32165.

**Reply by snalgae** [reply]
It is now #33871 and woo I support this idea

**Reply by Joehuu** [reply]
Has been added to tachyon release, closing to free up `Top: All`.
